sdk/examples/wallet/wallet.rs

```diff
@@ -1,11 +1,12 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! cargo run --example wallet --release
+//! TODO: Example description
```

Comment:
TODO

Reply:
You're telling me I made all these comments for no reason (except to annoy Alex)?